diff --git a/bin/opx-config-lag b/bin/opx-config-lag
index ca1bf85..6a19761 100755
--- a/bin/opx-config-lag
+++ b/bin/opx-config-lag
@@ -95,13 +95,6 @@ class cli(object):
         if cfg_obj is not None:
             self.helper_exit('LAG already exists')
 
-        # Assign mac-learn-mode
-        if self.args.learn_mode:
-            args_dict[attr_lag_learn_mode] = lag_learn_mode_reverse_map[self.args.learn_mode]
-
-        # Enable/Disable the LAG
-        args_dict[attr_intf_enable_state] = self.admin_status_flag
-
         self.cps_cb('create', args_dict)
 
 
@@ -116,14 +109,6 @@ class cli(object):
         self.set_member_ports(sorted(member_ports),'set')
 
 
-        # Assign unblocked ports
-        if len(self.unblocked_port_list) != 0:
-            self.cps_cb('set', 
-                        {attr_intf_name:self.args.name, 
-                         attr_lag_unblocked_ports:sorted(self.unblocked_port_list)
-                        }
-                        )
-
         # Assign blocked ports
         if len(self.blocked_port_list) != 0:
             self.cps_cb('set', 
@@ -132,6 +117,15 @@ class cli(object):
                         }
                        )
 
+        # Assign mac-learn-mode
+        if self.args.learn_mode:
+            args_dict[attr_lag_learn_mode] = lag_learn_mode_reverse_map[self.args.learn_mode]
+
+        # Enable/Disable the LAG
+        args_dict[attr_intf_enable_state] = self.admin_status_flag
+
+        self.cps_cb('set', args_dict)
+
 
     def delete(self):
         self.is_lag_present()
diff --git a/configure.ac b/configure.ac
index 1db4ed5..ba4e675 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2,7 +2,7 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.69])
-AC_INIT([opx-tools],[2.0.1+opx1],[ops-dev@lists.openswitch.net])
+AC_INIT([opx-tools],[2.0.1+opx2],[ops-dev@lists.openswitch.net])
 AM_INIT_AUTOMAKE([foreign subdir-objects])
 AC_CONFIG_MACRO_DIRS([m4])
 
diff --git a/debian/changelog b/debian/changelog
index d3853d9..57e64cc 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,9 @@
+opx-tools (2.0.1+opx2) unstable; urgency=medium
+
+  * Bugfix: slave ports not coming up when LAG is created through opx-config-lag
+
+ -- Dell EMC <ops-dev@lists.openswitch.net>  Fri, 18 Jan 2019 13:32:35 -0700
+
 opx-tools (2.0.1+opx1) unstable; urgency=medium
 
   * Bugfix: Added bond support to vlan util
